packages/venia-ui/lib/components/CheckoutPage/ShippingInformation/AddressForm/customerForm.js

```diff
@@ -134,6 +134,18 @@ const CustomerForm = props => {
         <Text type="hidden" field="default_shipping" initialValue={true} />
     );
 
+    const createErrorMessage = JSON.stringify(
+        errors.get('createCustomerAddressMutation')
+    );
+    const updateErrorMessage = JSON.stringify(
+        errors.get('updateCustomerAddressMutation')
+    );
+    const errorMessage = 'region_id is required for the specified country code';
```

Comment:
Testing for the specific error message from the admin seems very brittle (error messages could be translated and customised).

I would suggest something like this in region.js instead

```
const { validate, ...propsWithoutValidate } = regionProps;
```

and then 
```
    const regionField =
        regions.length || loading ? (
            <Select
                {...regionProps}
                field={fieldSelect}
                id={classes.root}
                items={regions}
            />
        ) : (
            <TextInput {...propsWithoutValidate} field={fieldInput} id={classes.root} optional="true" />
        );
```